.github/workflows/ci.yaml

```diff
@@ -155,15 +159,15 @@ jobs:
           # converts manifest yaml to a json object of {SOFTWARE_NAME: URL#REF, ...}
           urlrefs=$(
             cat .github/container/manifest.yaml |\
-            yq -o=json 'to_entries | .[] | select(.value.mode == "git-clone") | {( .key | upcase | sub("-", "_") ): .value.url + "#" + .value.latest_verified_commit}' |\
+            yq -o=json 'to_entries | .[] | select(.value.mode == "git-clone") | {( .key | upcase | sub("-", "_") ): .value.url + "#" + .value.commit}' |\
```

Comment:
It was a bit suprising to me, that I had to add `mode: git-clone` in order to pull a non-HEAD commit until I found this line. Maybe some documentation about `manifest.yaml` would be helpful at some point but no high prio as most devs here are familiar with its mechanics I suppose

Reply:
I'm not sure if the mode is still needed at all. But needing better documentation is an evergreen comment...